go/libraries/doltcore/sqle/table_editor.go

```diff
@@ -380,3 +281,98 @@ func (te *tableEditor) flush(ctx *sql.Context) error {
 	te.t.table = newTable
 	return te.t.db.SetRoot(ctx, newRoot)
 }
+
+func (te *tableEditor) updateIndexes(ctx *sql.Context, tbl *doltdb.Table, originalRowData types.Map, updated types.Map) (*doltdb.Table, error) {
+	if len(te.indexEds) == 0 {
+		return tbl, nil
+	}
+
+	for _, key := range te.affectedKeys {
```

Comment:
I notice that affectedKeys isn't written to for Delete. Won't that mean we don't update indexes when a row is deleted?

Reply:
It's there, I just you just accidentally overlooked it lol
```go
func (te *tableEditor) Delete(ctx *sql.Context, sqlRow sql.Row) error {
	dRow, err := SqlRowToDoltRow(te.t.table.Format(), sqlRow, te.t.sch)
	if err != nil {
		return err
	}

	key, err := dRow.NomsMapKey(te.t.sch).Value(ctx)
	if err != nil {
		return errhand.BuildDError("failed to get row key").AddCause(err).Build()
	}
	hash, err := key.Hash(dRow.Format())
	if err != nil {
		return err
	}

	delete(te.addedKeys, hash)
	te.removedKeys[hash] = key
	te.affectedKeys[hash] = key

	if te.ed == nil {
		te.ed, err = te.newMapEditor(ctx)
		if err != nil {
			return err
		}
	}

	te.ed = te.ed.Remove(key)
	return nil
}
```